backend.native/tests/interop/incomplete_types/main.kt

```diff
@@ -0,0 +1,8 @@
+import library.*
+import kotlinx.cinterop.*
+
+fun main() {
+    println(s.ptr)
+    println(u.ptr)
+    println(array)
+}
```

Comment:
Testing actual contents would be great.

Reply:
By contents I mean contents of the global, e.g. `s.name` (can be obtained with a dedicated getter function) :)
The contents should also be non-trivial (i.e. not null).

Reply:
The test still doesn't actually check that `s`, `u` and `array` represent the original C globals, because it doesn't examine the contents of the globals in Kotlin before it overwrites them.
For example, getters could return pointers to wrong parts of the memory, and the test wouldn't detect this.